### DISCUSSION TITLE
How Magnetic radius parameter works?

### DISCUSSION THREAD

**Discussion by iberck** [pubpub-zz/ppInk#225] [Q&A] [answered]
In the application options, how do you use the "magnetic radius (%src)" parameter?
I have changed the parameter multiple times with negative, positive, decimal and integer values and the behavior is the same.

Does anyone know what this parameter is for?

Thank you

**Comment by pubpub-zz** [comment, marked as answer]
the magnetic effect applies when line, arrow, rectangle... are selected not for free hand drawing
when the magnetic effect is activated the end point of the arrow will try to stick to the existing strokes (even to handfree strokes) already drawn next to  the cursor. the bigger the value will be the further the cursor can be before the magnetic effect engages
About the negative value, a regression may have occur will check that tomorrow
